node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
+    'no-tabs': error,
+    'spaced-comment': [ error, always ],
+    'array-callback-return': warn,
+    'eqeqeq': [ error, always, { null: 'ignore' } ],
+    'no-else-return': [ warn, { allowElseIf: false } ],
+    'yoda': error,
+    'no-throw-literal': error,
+    'no-self-compare': error,
+    'no-return-assign': error,
```

Comment:
Don't allow variable assignment in the return statement. It can be hard to understand what value is actually being returned.
https://eslint.org/docs/rules/no-return-assign

```js
let foo = 1
// Old
function doSomething() {
  return foo = bar + 2;
}
// New
function doSomething() {
  foo = bar + 2
  return foo
}
```
*Not auto-fixable*

Reply:
does this prevent mutation full stop? that's interesting.